gnd/src/main/java/com/google/android/gnd/persistence/local/room/dao/TileDao.java

```diff
@@ -27,7 +28,7 @@
 public interface TileDao extends BaseDao<TileEntity> {
 
   @Query("SELECT * FROM tile")
-  Single<List<TileEntity>> findAll();
+  Flowable<List<TileEntity>> findAll();
```

Comment:
Why return a Flowable here? Do we want to subscribe to future tile changes?

Reply:
The corresponding method in `LocalDataStore` was unused. But I think @scolsen would be needing that later since he added them. I've added tests for the same to verify that it indeed works as it says.

Reply:
> The semantics of Single and Flowable are different in Room generated classes; Single returns a result and completes, whereas Flowable continues emitting values on each change. We convert to Flowable in upstream code because we're merging with a hot observable that doesn't complete as soon as a value is emitted. Here we expect the stream to terminate after the first item is emitted, so Single is appropriate here.

Reverting back Flowable to Single fails the test testGetTilesOnceAndStream. @gino-m Can you or @scolsen please take a look at the test to verify that I am testing the correct behavior or not?